src/components/FloatingDropdown/FloatingDropdown.tsx

```diff
@@ -103,6 +107,16 @@ export class FloatingDropdown extends React.Component<FloatingDropdownProps> {
     };
   }
 
+  private readonly onExpandedChange = (isExpanded: boolean) => {
```

Comment:
For private function, we usually add the prefix _.
Can you please change it for :
`_onExpandedChange(isExpanded)`